tap_mysql/__init__.py

```diff
@@ -202,7 +202,10 @@ def schema_for_column(c):
     result = Schema(inclusion=inclusion, selected=True)
     result.sqlDatatype = c.column_type
 
-    if t in BYTES_FOR_INTEGER_TYPE:
+    if t == 'bit' or c.column_type == 'tinyint(1)':
```

Comment:
Is it possible that the column type could be a different string case? Should we do case-insensitive comparison?

Reply:
@mdelaurentis yeah we should do case insensitive comparison here -- good call